subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/report/listener/AvitoFileStorageUploader.kt

```diff
@@ -0,0 +1,45 @@
+package com.avito.instrumentation.internal.report.listener
+
+import com.avito.android.Result
+import com.avito.filestorage.RemoteStorage
+import com.avito.report.model.Entry
+import okhttp3.HttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import java.io.File
+
+internal class AvitoFileStorageUploader(
+    private val remoteStorage: RemoteStorage
+) : TestArtifactsUploader {
+
+    override suspend fun uploadLogcat(logcat: String): Result<HttpUrl> {
```

Comment:
Generic uploader shouldn't know about exact use cases.
Let's express this as string\log uploading.